config.py

```diff
@@ -13,3 +17,16 @@ class Config(object):
     SESSION_COOKIE_HTTPONLY = True
     SESSION_COOKIE_SECURE = True
     API_HOSTNAME = os.environ.get("API_HOSTNAME")
+
+    USE_LOCAL_DATA = strtobool(os.getenv("USE_LOCAL_DATA", "True"))
+
+    # Funding Service Design Post Award
+    FSD_USER_TOKEN_COOKIE_NAME = "fsd_user_token"
+    AUTHENTICATOR_HOST = os.environ.get("AUTHENTICATOR_HOST", "authenticator")
+    SESSION_COOKIE_DOMAIN = os.environ.get("SESSION_COOKIE_DOMAIN")
+    COOKIE_DOMAIN = os.environ.get("COOKIE_DOMAIN", None)
+
```

Comment:
Wonder if we need a SECRET_KEY or at least on deployed environments as per https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY else I think we may get confusing behaviour between environments.